components/docs-chef-io/content/automate/chef_infra_in_chef_automate.md

```diff
@@ -25,6 +24,13 @@ The list of parameters are as follows:
 | nginx['ssl_protocols']                          | TLSv1.2                                   | TLSv1.2                             | ```[cs_nginx.v1.sys.ngx.http]```<br>```ssl_protocols="TLSv1.2"```      |
 | nginx['worker_connections']                     | 10240                                     | 10240                               | ```[cs_nginx.v1.sys.ngx.events]```<br>```worker_connections=10240``` |
 | nginx['worker_processes']                       | 4                                         | 2                                   | ```[cs_nginx.v1.sys.ngx.main]```<br>```worker_processes=4```     |
+| nginx['gzip']                                   | ON                                        |ON                                   |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip= "on"```              |
+| nginx['gzip_comp_level']                         |2                                          |2                                    |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_comp_level=2```       |
+| nginx['gzip_http_version']                       |1                                          |1                                    |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_http_version=1```     |
+| nginx['gzip_types']                              | Refer  [nginx[gzip_type]](https://docs.chef.io/server/config_rb_server_optional_settings/#nginx-14) | Refer  [nginx[gzip_type]](https://docs.chef.io/server/config_rb_server_optional_settings/#nginx-14)  |```[cs_nginz.v1.sys.ngx.http]```<br>```gzip_types ="[ 'text/plain]"```            |
```

Comment:
missing single quote at the end for `text/plain`

Reply:
Hey @Dmaddu, Thank you for pointing out these, Fixing this right away.